lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -41,13 +66,21 @@
                        var self = this;
                        this.$el.html(this.template({}));
                        _.each(this.tabs, function(tabInfo, index) {
-                           var tabEl = $(_.template(tabTemplate, {
-                               index: index,
-                               title: tabInfo.title,
-                               url: tabInfo.url
-                           }));
+                           var tabId = tabInfo.url,
+                               tabEl = $(_.template(tabTemplate, {
+                                   index: index,
+                                   title: tabInfo.title,
+                                   url: tabInfo.url,
+                                   tabPanelId: tabId
+                               })),
+                               tabContainerEl = this.$('.tabs');
                            self.$('.page-content-nav').append(tabEl);
-                       });
+
+                           // Handle tab panels
```

Comment:
Nice catch. I must have left that from development - I like to leave myself comment notes sometimes. I'll make this more descriptive.